tests/10apidoc/33room-members.pl

```diff
@@ -35,23 +35,21 @@
          $body->{room_id} eq $room_id or
             die "Expected 'room_id' to be $room_id";
 
-         Future->done(1);
-      });
-   },
-
-   check => sub {
-      my ( $user, $room_id, undef ) = @_;
-
-      matrix_get_room_state( $user, $room_id,
-         type      => "m.room.member",
-         state_key => $user->user_id,
-      )->then( sub {
-         my ( $body ) = @_;
-
-         $body->{membership} eq "join" or
-            die "Expected membership to be 'join'";
-
-         Future->done(1);
+         # Retry getting the state a few times, for it may take some time to
```

Comment:
this reads really awkwardly.

```suggestion
         # Retry getting the state a few times, as it may take some time to
```
or 'since'?

Reply:
this still says 'for' ?